Sources/Core/ObjectiveCIR.swift

```diff
@@ -246,11 +246,26 @@ public struct ObjCIR {
     }
 
     static func enumStmt(_ enumName: String, body: () -> [String]) -> String {
+        let enums = body()
+        let enumType : String
+        switch enums.count {
```

Comment:
After I get the basics for this diff working, I'll have to update this to handle if there are say, 2 enumeration values of 1000 and 1001. Those will not fit into an unsigned char.